docs/content/doc/usage/backup-and-restore.en-us.md

```diff
@@ -80,6 +80,14 @@ mysql --default-character-set=utf8mb4 -u$USER -p$PASS $DATABASE <gitea-db.sql
 service gitea restart
 ```
 
+### Restore Database in Docker
+
+```
+docker-compose up db # do not run `docker-compose up`, which will init new db from scratch.
```

Comment:
is this really needed? This will not work in swarm

Reply:
The main point is init db first and then run gitea container and db container. Otherwise, gitea will create a new db and init the tables.  The doc will save time for those who use docker-compose.